scripts/gen-netatt.sh

```diff
@@ -238,12 +238,23 @@ spec:
       "cniVersion": "0.3.1",
       "name": "octavia",
       "type": "macvlan",
+EOF_CAT
+if [ -n "$IPV4_ENABLED" ]; then
+    cat >> ${DEPLOY_DIR}/octavia.yaml <<EOF_CAT
       "master": "${INTERFACE}.$((${VLAN_START}+${VLAN_STEP}*3))",
       "ipam": {
         "type": "whereabouts",
         "range": "172.23.0.0/24",
         "range_start": "172.23.0.30",
         "range_end": "172.23.0.70"
+EOF_CAT
+elif [ -n "$IPV6_ENABLED" ]; then
+    cat >> ${DEPLOY_DIR}/octavia.yaml <<EOF_CAT
+        "range": "fd00:eeee::/64",
```

Comment:
The test log shows an syntax error. https://logserver.rdoproject.org/42/742/4f1fd13a5647fb8e709451e178d0eb72706eea3c/github-check/install-yamls-crc-podified-edpm-baremetal/521adc3/job-output.txt

It looks like you need at least the line `"ipam": {` here as well

Reply:
Thanks!